### ISSUE TITLE
Set mongodb location

### ISSUE THREAD

**Issue by michaelb-01** [wojtkowiak/meteor-desktop#84]
How can I set the location of the mongo database? 
I would like to it to a windows network drive so that everyone in the office has access to it

**Comment by wojtkowiak** [comment]
This integration builds only the meteor client part so it is up to you where you will host your meteor server - no limitations. While building you can either pass the url to meteor with `--mobile-server` or directly to meteor-desktop where the first parameter is the url - i.e. `npm run desktop -- build-installer <ddp_url>`

**Comment by michaelb-01** [comment]
Thanks for your reply. Sorry for being slow but i'm still not clear how to achieve what i'm after.
Say we have 20 computers on a shared windows network drive, how do I set the mongo database to that drive, e.g. S:/mongodb? (I understand this can be set with mobile-server or ddp_url)

**Comment by michaelb-01** [comment]
Maybe this is more a meteor specific question but i'm still really stuck on this and would really like someone to show me exactly what to do..

Do I need to set the MONGO_URL? or set the --dbpath? Or use ground_db?

**Comment by banjerluke** [comment]
This is not really meteor-desktop related so we shouldn't discuss it here, but I'll give you a couple hints at least...

The `MONGO_URL` doesn't point to a filepath. It points to the URL of a running mongo server. (Fun fact: MongoDB needs to use a database located on a local disk drive, not a network drive. This is why you can't run `meteor` on a meteor app that's sitting in a shared network folder.)

So if you had a mongo server running on one of your computers, say one with an IP address of `192.168.1.123`, then you could set the [MONGO_URL environment variable](https://docs.meteor.com/environment-variables.html) (`set MONGO_URL=mongodb://192.168.1.123:<port-number>` in Windows) before running `meteor`, and Meteor will connect to that database instead of the local one in `.meteor/db`. This is assuming you're in development of course... I don't see why anyone in their right mind would attempt to deploy to production on a Windows machine. 😉

**Comment by michaelb-01** [comment]
Hey @banjerluke, thanks for your reply. Unfortunately I don't think its working..

I'm currently developing on Mac (but I will deploying on Windows), so I tried:

`set MONGO_URL=mongodb://localhost/data/db_meteor:3001
`
I then run meteor, the app starts fine. In a new shell I type meteor mongo and it says:

`connecting to: 127.0.0.1:3001/meteor`

So I think its still using the using the local one in .meteor/db?

I thought to set variables I had to use 'export' so I tried:

`export MONGO_URL=mongodb://localhost/data/db_meteor:3001`

However when I run _meteor_ after that I get:

`Error: double colon in host identifier`

Any ideas what i'm doing wrong?

(I tried replacing localhost with my ip address and I get the same results)
